client/src/components/game/phases/Events.vue

```diff
@@ -1,32 +1,33 @@
 <template>
-  <div class="events-container tour-event">
-    <div class="event-deck-container">
-      <div class="section-text">
-        <p>Event Deck</p>
-      </div>
-      <div class="events">
-        <EventCard
-          class="event-container"
-          v-for="(event, index) in eventsForTheRound"
-          :key="index"
-          :event="event"
-          :visible="eventVisible(index)"
-          :active="eventActive(index)"
-        />
-      </div>
-    </div>
-    <div class="active-event-container tour-active-events">
-      <div class="section-text">
-        <p>Active Events</p>
-      </div>
-      <div class="event-holder">
-        <div class="event-info">
-          <div class="event-title">
-            <p>Event #{{ eventsProcessed + 1 }}- {{ currentEvent.name }}</p>
+  <div class="c-events container tour-event">
+    <div class="wrapper row">
+      <div class="eventdeck col-4">
+        <div class="topbar">
+          <p class="title">Event Deck</p>
+        </div>
+        <div class="outer-wrapper">
+          <div class="wrapper event-scroll">
+            <EventCard
+              class="event-container"
+              v-for="(event, index) in eventsForTheRound"
+              :key="index"
+              :event="event"
+              :visible="eventVisible(index)"
+              :active="eventActive(index)"
+            />
           </div>
         </div>
-        <div class="event-action-wrapper">
-          <EventContainer :event="currentEvent" :key="eventNumber" />
+      </div>
+      <div class="activeevents col-8 tour-active-events">
```

Comment:
minor nitpick and not blocking this merge but we should refactor our CSS classes to be consistently kebab cased, e.g., `active-events`